CHANGELOG.md

```diff
@@ -1,5 +1,31 @@
 # CHANGELOG
 
+## 6.0.0
+
+* `geoipupdate` now supports configuration via environment variables. Any
+  configuration set this way will override any value from the config file,
+  but still be overridden by any associated command line option (if any).
+  The following new environment variables are supported:
+
+    * `GEOIPUPDATE_ACCOUNT_ID`
+    * `GEOIPUPDATE_CONF_FILE`
+    * `GEOIPUPDATE_DB_DIR`
+    * `GEOIPUPDATE_EDITION_IDS`
+    * `GEOIPUPDATE_HOST`
+    * `GEOIPUPDATE_LICENSE_KEY`
```

Comment:
Yeah it is hard to say. I agree with the idea, though I didn't think much about it here as I've used something like the following in some CI workflows more than once:
```
environment:
    GITHUB_ACCESS_TOKEN:
        secure: EPqbi0m6WbH/4m84G1mHPyJ9jf9J90jFw23raWJC1Js62Owlmww0fmexoVEp
```

Reply:
Yeah, it does seem like a lot of tools take secrets directly like that. Maybe it's ok. Internally we're trying not to do it as a best practice I believe, and since this docker image currently avoids it I'm leaning to thinking we should continue avoiding it.